src/sentry/static/sentry/app/views/organizationDashboard/data/queries/anonymousUsersAffected.jsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * Anonymous affected users
+ */
+const anonymousUsersAffectedQuery = {
+  fields: [],
+  conditions: [['user.email', 'IS NULL', null]],
+  aggregations: [['count()', null, 'Anonymous Users']],
+  limit: 1000,
+
+  orderby: '-time',
+  groupby: ['time'],
+  rollup: 86400,
+};
```

Comment:
Is this the shape of the data we're thinking of storing in the dashboard widget tables?

Reply:
Yeah, this is the same shape as discover queries